packages/bundler-plugin-core/src/types.ts

```diff
@@ -104,6 +104,23 @@ export interface Options {
 
   /** Override values for passing custom information to API. */
   uploadOverrides?: UploadOverrides;
+
+  sentry?: {
+    /** Only send bundle stats to sentry (used within sentry bundler plugin). */
+    sentryOnly?: boolean;
+
+    /** Enables stats to be sent to sentry, set this to true on release */
+    isEnabled?: boolean;
```

Comment:
Can you add in the JSDoc comment about the default value here, just to keep the user informed 👀

Reply:
👍